lang/guile-3.0/Portfile

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                guile-3.0
+categories          lang
+maintainers         wolfsden.cz:~+macports
+license             LGPL-2.1+
+
+# current version
+version             3.0.10
+revision            0
+checksums           rmd160  c42990081a8fffaf63874d16f457d8e50db8bc36 \
+                    sha256  2dbdbc97598b2faf31013564efb48e4fed44131d28e996c26abe8a5b23b56c2a \
+                    size    9738824
+
+compiler.c_standard 2011
+
+universal_variant   no
+# Never cross compile.
+supported_archs     ${build_arch}
+
+description         GNU's Ubiquitous Intelligent Language for Extension (guile)
+long_description    \
+        Guile is an interpreter for the Scheme programming language, packaged \
+        for use in a wide variety of environments. Guile implements Scheme as \
+        described in the Revised^5 Report on the Algorithmic Language Scheme  \
+        (usually known as R5RS), providing clean and general data and control \
+        structures.                                                           \
+                                                                              \
+        Guile goes beyond the rather austere language presented in R5RS,      \
+        extending it with a module system, full access to POSIX system calls, \
+        networking support, multiple threads, dynamic linking, a foreign      \
+        function call interface, powerful string processing, and many other   \
+        features needed for programming in the real world.
+
+distname            guile-${version}
+homepage            https://www.gnu.org/software/guile/
+master_sites        https://ftp.gnu.org/gnu/guile/
+
+# pkgconfig: guile-config is a wrapper around pkg-config
+depends_lib-append \
+                    port:boehmgc \
+                    port:gettext \
+                    port:gmp \
+                    port:gnutls \
+                    port:libffi \
+                    port:libiconv \
+                    port:libtool \
+                    port:libunistring \
+                    port:ncurses \
+                    port:pkgconfig \
```

Comment:
Perhaps, rather `path:bin/pkg-config:pkgconfig`, unless it is known not to work with `pkgconf`.